busybox.yaml

```diff
@@ -15,8 +17,12 @@ package:
         - /usr/bin
         - /usr/sbin
       script: |
-        #!/bin/busybox sh
-        /bin/busybox --install -s
+        #!/usr/bin/busybox sh
+        for i in `/usr/bin/busybox --list`; do
+          [ -e /usr/bin/$i ] && continue
+          [ -h /usr/bin/$i ] && continue
+          /usr/bin/busybox ln -s /bin/busybox /usr/bin/$i
+        done
```

Comment:
Why /bin/busybox here?

Also, relative would be nice.

Reply:
Because that is what currently apko generates See https://github.com/search?q=repo%3Achainguard-dev%2Fapko%20bin%2Fbusybox&type=code

Once this lands; we can work on fixing apko to start generating usr-merged layout; and likely relative links too. For now, I'm trying to ensure these upgrades get one the same layout & contents as the fresh apko build as much as possible - including these choices of paths.

Reply:
makes sense, but isn't easily discoverable. Could you note that in a comment?